### PR TITLE
Development

### PR DESCRIPTION
**Pull request by quac88**
## Description of what you changed
Created a Poker module complete with unit testing and a example run.py file for gameplay. Note: The Game class still requires unit testing
KNOWN BUGS: When playing hands, the game may crash if a player runs out of chips. Players may have issues going all in when they cannot cover the size of the current bet. pot.total may not update correctly if a player goes all in to a bet the player cannot fully cover.

## Did you add any new dependencies ex (using a new library, python version, etc)?
N/A

## How did you test your changes?
Yes. See the tests folder. Note: Game class is tested, but not fully tested with unit tests

## What issue (if any) does this fix?
N/A

## Checklist:
- [x] I have read the [contributing.md](contributing.md) rules
- [x] I am follwoing the [branch naming rules](contributing.md#format-for-branch-names)
- [x] I am using symantic release and angular style commits (see [commits.md](docs/commits.md))
- [x] I have tested my changes
- [x] I have added any new dependencies to the [README.md](README.md)
- [x] I have added any new dependencies to the [requirements.txt](requirements.txt)

### CODE REVIEW

**Review by quac88 — COMMENTED** [review]
LGTM

File: requirements.txt
Comment (MichaelAGolden) [inline requirements.txt:1]: adding tqdm

File: tests/test_cardecky.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'src', 'poker')))
```
Comment (MichaelAGolden) [inline tests/test_cardecky.py:3]: added import os, sys, and abspath import for test file

File: tests/test_cardecky.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+import unittest
+from unittest.mock import patch
+from itertools import product
+from cardecky import Card, Deck, Rank, Suit, HandRanker
```
Comment (MichaelAGolden) [inline tests/test_cardecky.py:7]: Removed relative import naming on cardecky

File: tests/test_game.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
+import unittest
+from unittest.mock import Mock, patch
+import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'src', 'poker')))
+from cardecky import Deck, HandRanker, Rank, Suit
+from game import Player, Pot, Dealer, Table, PlayerAction, Game
```
Comment (MichaelAGolden) [inline tests/test_game.py:7]: removed relative import path on game and cardecky

**Review by MichaelAGolden — APPROVED** [review]
Added TQDM to requirements.txt and changed import path to absolute using @quac88's method to both test files for consistency. Tested and working as intended

**Review by MichaelAGolden — APPROVED** [review]
lgtm

**Pull request merged**